src/serveses/debounce.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/ban-types
+export const debounce = (callback: Function, delay: number) => {
```

Comment:
Avoid using the Function type. Consider using a stricter function type that describes the function's parameters and return type.